libs/ui/src/components/NeoButton/NeoButton.vue

```diff
@@ -39,6 +40,7 @@ withDefaults(
     active?: boolean
     fixedWidth?: boolean
     noShadow?: boolean
+    flat?: boolean
```

Comment:
1. depending on the required looks in hover, you may be able to use `text` variant
2. if not - then plz change the prop name to `noBorder`